safe/tokenizer.py

```diff
@@ -136,13 +136,33 @@ def bos_token_id(self):
 
     @property
     def pad_token_id(self):
-        """Get the bos token id"""
+        """Get the pad token id"""
         return self.tokenizer.token_to_id(self.tokenizer.pad_token)
 
     @property
     def eos_token_id(self):
-        """Get the bos token id"""
+        """Get the eos token id"""
         return self.tokenizer.token_to_id(self.tokenizer.eos_token)
+    
+    @property
+    def unk_token_id(self):
+        """Get the unk token id"""
+        return self.tokenizer.token_to_id(self.tokenizer.unk_token)
+    
+    @property
+    def mask_token_id(self):
+        """Get the mask token id"""
+        return self.tokenizer.token_to_id(self.tokenizer.mask_token)
+    
+    @property
+    def cls_token_id(self):
```

Comment:
For the default tokenizer,  our convention is:

`cls = bos` and `sep = eos`

 So if the cls_token and sep_token are not defined, that's what you should return.
 
 Also remember that you can always use `tokenizer.get_pretrained()`  to get an huggingface PreTrainedTokenizer object

Reply:
Got it. Thanks :-)